src/test/Makefile

```diff
@@ -63,4 +63,8 @@ check_dirs := regress isolation2
 installcheck-cbdb-parallel: install
     $(call recurse,installcheck-cbdb-parallel,$(check_dirs))
 
+singlenode_check_dirs := singlenode_regress isolation singlenode_isolation2
+installcheck-singlenode: install
```

Comment:
need a singlenode ci.

Reply:
I have already talked to @sandiandian , he'll take care of it after the code is merged. I'd like to use his help. This pr really makes me exhausted.